Doppler.Sap/Models/DopplerUserDTO.cs

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
I think that the name of this file should be the same that the name of the class `DopplerUserDto`

Reply:
Thanks Naza! I changed it on last commit.